### ISSUE TITLE
SWHS: Missing Notes in Data Constraints Table

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#1490]
Related to #26 - the asterisks, etc. for the notes are missing in the Input Data Constraints Table:
![idctable](https://user-images.githubusercontent.com/35857611/58911775-96ac8700-86e6-11e9-8b36-81b139c85a4e.png)

@bmaclach You made the original #26 - what's the status of that? Can we add asterisks, etc. to tables?

**Comment by bmaclach** [comment]
As far as I know, there is still no way to add notes to tables in Drasil.

**Comment by samm82** [comment]
Yikes - OK, this might be a spicier issue than I thought it would be. Should I move this information to the original #26 issue and close this? @smiths @bmaclach

**Comment by bmaclach** [comment]
@samm82 I think it's okay to keep this issue open for making the changes in SWHS, whereas the original issue is for implementing the feature of having notes in tables.

**Comment by smiths** [comment]
I agree with @bmaclach.  We should keep this issue, so that we don't lose the information from the manual SWHS example.  I'm sure there are other ways to document the information other than notes attached to the table.  We might come up with a solution in that direction sooner than adding notes to tables.

**Comment by balacij** [comment]
I suppose this is related to #26 from @bmaclach's comment? I'm not exactly sure what the 'problem' is that this proposed solution solves.
